masonite/app.py

```diff
@@ -223,6 +223,9 @@ def _find_annotated_parameter(self, parameter):
         raise ContainerError(
             'The dependency with the {0} annotation could not be resolved by the container'.format(parameter))
 
+    def get_parameters(self, obj):
```

Comment:
Where this will get used?

Reply:
just updated

Reply:
:)

Reply:
oh lol, I jumped in the middle of your work.

Reply:
np lol. I was actually done but we caught that at the exact same time and I updated PR